template/test-report.vuln-card.hbs

```diff
@@ -12,6 +12,13 @@
             <div class="label label--{{metadata.severity}}">
                 <span class="label__text">{{metadata.severity}} severity</span>
             </div>
+            {{#if metadata.exploitMaturity}}
+            {{#if_not_eq metadata.exploitMaturity "Not Defined"}}
```

Comment:
nit: if you make the `metadata.exploitMaturity` a union type that could be `null` or `undefined`, then you could remove the second if statement here.

If you're going to keep the second condition, can you combine them into a single statement (e.g., `if metadata.exploitMaturity && metadata.exploitMaturity !== "Not Defined"`). This might not be possible -- I'm not familiar with handlebars

Reply:
I agree with @bgardiner here, we could simplify the template by allowing null values. From my understanding we are falling back to "Not Defined" in `getExploitMaturity` method, only to use it for preventing the render of an Exploit Maturity element in the report. 

We should also extend the test scenarios to verify this behaviour and have an assertion that the label--exploit element is not present when exploitMaturity data does not exist on the vulnerability.